__tests__/endpoints.test.js

```diff
@@ -96,3 +96,43 @@ describe("/api", () => {
       });
   });
 });
+
+describe('/api/articles/:article_id', () => {
+    test('test for 200 status on completion', () => {
+        return request(app).get(`/api/articles/3`).expect(200);
+    });
+    test('should have the correct article by id', () => {
+        return request(app).get(`/api/articles/1`).expect(200).then((result) => {
+            expect(result.body).toEqual([{
+                article_id : 1,
+                title: 'Living in the shadow of a great man',
+                topic: 'mitch',
+                author: 'butter_bridge',
+                body: 'I find this existence challenging',
+                created_at: '2020-07-09T20:11:00.000Z',
+                votes: 100,
+                article_img_url:
+                  'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+              }])
+        });
+    });
+    test('should have all 8 properties', () => {
+      return request(app).get(`/api/articles/1`).expect(200).then((result) => {
+        expect(result.body).toEqual(
+          expect.arrayContaining([
+            expect.objectContaining({
+              article_id : 1,
+                title: 'Living in the shadow of a great man',
+                topic: 'mitch',
+                author: 'butter_bridge',
+                body: 'I find this existence challenging',
+                created_at: '2020-07-09T20:11:00.000Z',
+                votes: 100,
+                article_img_url:
+                  'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+            })
+          ])
+        )
+      })
+    });
+});
```

Comment:
There are no sad path tests. What happens If I make a request like /api/articles/999999 ? would it be a different error to /api/articles/not_an_id ?